### PR TITLE
Feature/error handling

### PR DESCRIPTION
**Pull request by KatyaBaranovskaya**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,8 @@
     "bootstrap": "^4.3.1",
     "classnames": "^2.2.6",
     "event-emitter": "^0.3.5",
+    "jsonwebtoken": "^8.5.1",
```
Comment (andreymikhadyuk) [inline package.json]: Is this library used somewhere?

File: src/pages/ErrorPage/container.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      error: null,
+    };
+  }
+
+  componentDidMount() {
+    const queryParams = queryString.parse(this.props.location.search);
```
Comment (andreymikhadyuk) [inline src/pages/ErrorPage/container.jsx]: I would move this logic to `constructor`

File: src/pages/LoginPage/container.jsx [hunk 3]
```diff
@@ -33,22 +34,29 @@ class LoginPage extends React.Component {
...
       .catch((error) => {
         console.log(error);
-        this.setState({ isLoading: false });
+        const newState = { isLoading: false };
+        if (error.response.status === 401) {
+          newState.error = 'Credentials incorrect, please check login and password';
+        } else {
+          navigate(`/error?error=${error.response.data.message}`);
+        }
+        this.setState(newState);
```
Comment (andreymikhadyuk) [inline src/pages/LoginPage/container.jsx]:
```
console.log(error);

if (error.response.status !== 401) {
  return navigate(`/error?error=${error.response.data.message}`);
}

this.setState({
  isLoading: false,
  error: 'Credentials incorrect, please check login and password',
});
```

**Review by andreymikhadyuk — CHANGES_REQUESTED** [review]

File: src/pages/ErrorPage/container.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import queryString from 'query-string'
+
+import ErrorPageView from './view';
+
+class ErrorPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const queryParams = queryString.parse(this.props.location.search);
```
Comment (andreymikhadyuk) [inline src/pages/ErrorPage/container.jsx]:
```suggestion
    const queryParams = queryString.parse(props.location.search);
```

File: src/pages/ErrorPage/container.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    const { error } = queryParams;
+
+    this.state = {
+      error: error ? error : null,
```
Comment (andreymikhadyuk) [inline src/pages/ErrorPage/container.jsx]:
```suggestion
      error: error || null,
```

**Review by andreymikhadyuk — CHANGES_REQUESTED** [review]

File: src/pages/ErrorPage/container.jsx (hunk 4 above)
Comment (KatyaBaranovskaya) [reply src/pages/ErrorPage/container.jsx]: ))))))))))))))))))))))

**Review by andreymikhadyuk — APPROVED** [review]
👍

**Pull request merged**